src/openrct2-ui/windows/Finances.cpp

```diff
@@ -87,79 +87,79 @@ enum
-    { WWT_TAB,      1,  65,     95,     17, 43,     IMAGE_TYPE_REMAP | SPR_TAB, STR_FINANCES_SHOW_PARK_VALUE_TAB_TIP    }, \
-    { WWT_TAB,      1,  96,     126,    17, 43,     IMAGE_TYPE_REMAP | SPR_TAB, STR_FINANCES_SHOW_WEEKLY_PROFIT_TAB_TIP }, \
-    { WWT_TAB,      1,  127,    157,    17, 43,     IMAGE_TYPE_REMAP | SPR_TAB, STR_FINANCES_SHOW_MARKETING_TAB_TIP     }, \
-    { WWT_TAB,      1,  158,    188,    17, 43,     IMAGE_TYPE_REMAP | SPR_TAB, STR_FINANCES_RESEARCH_TIP               }
+    MakeRemapWidget({  3, 17}, {31, 27}, WWT_TAB, 1, SPR_TAB, STR_FINANCES_SHOW_SUMMARY_TAB_TIP      ), \
+    MakeRemapWidget({ 34, 17}, {31, 27}, WWT_TAB, 1, SPR_TAB, STR_FINANCES_SHOW_CASH_TAB_TIP         ), \
+    MakeRemapWidget({ 65, 17}, {31, 27}, WWT_TAB, 1, SPR_TAB, STR_FINANCES_SHOW_PARK_VALUE_TAB_TIP   ), \
+    MakeRemapWidget({ 96, 17}, {31, 27}, WWT_TAB, 1, SPR_TAB, STR_FINANCES_SHOW_WEEKLY_PROFIT_TAB_TIP), \
+    MakeRemapWidget({127, 17}, {31, 27}, WWT_TAB, 1, SPR_TAB, STR_FINANCES_SHOW_MARKETING_TAB_TIP    ), \
+    MakeRemapWidget({158, 17}, {31, 27}, WWT_TAB, 1, SPR_TAB, STR_FINANCES_RESEARCH_TIP              )
 
 static rct_widget _windowFinancesSummaryWidgets[] =
 {
     WINDOW_SHIM(WINDOW_TITLE, WW_OTHER_TABS, WH_SUMMARY),
-    { WWT_RESIZE,           1,  0,      529,    43, WH_SUMMARY - 1,    0xFFFFFFFF,                 STR_NONE                                },
+    MakeWidget({0, 43}, {530, 266}, WWT_RESIZE, 1),
     TAB_WIDGETS,
-    { WWT_SCROLL,           1,  130,    520,     50,    260,    SCROLL_HORIZONTAL,                  STR_NONE },
-      SPINNER_WIDGETS      (1,  64,     160,    279,    292,    STR_FINANCES_SUMMARY_LOAN_VALUE,    STR_NONE), // NB: 3 widgets.
+    MakeWidget        ({130,  50}, {391, 211}, WWT_SCROLL,  1, SCROLL_HORIZONTAL              ),
+    MakeSpinnerWidgets({ 64, 279}, { 97,  14}, WWT_SPINNER, 1, STR_FINANCES_SUMMARY_LOAN_VALUE), // NB: 3 widgets.
     { WIDGETS_END },
 };
 
 static rct_widget _windowFinancesCashWidgets[] =
 {
     WINDOW_SHIM(WINDOW_TITLE, WW_OTHER_TABS, WH_OTHER_TABS),
-    { WWT_RESIZE,           1,  0,      529,    43,     WH_OTHER_TABS - 1,    0xFFFFFFFF,             STR_NONE },
+    MakeWidget({0, 43}, {530, 214}, WWT_RESIZE, 1),
     TAB_WIDGETS,
     { WIDGETS_END },
 };
 
 static rct_widget _windowFinancesParkValueWidgets[] =
 {
     WINDOW_SHIM(WINDOW_TITLE, WW_OTHER_TABS, WH_OTHER_TABS),
-    { WWT_RESIZE,           1,  0,      529,    43,     WH_OTHER_TABS - 1,    0xFFFFFFFF,             STR_NONE },
+    MakeWidget({0, 43}, {530, 214}, WWT_RESIZE, 1),
     TAB_WIDGETS,
     { WIDGETS_END },
 };
 
 static rct_widget _windowFinancesProfitWidgets[] =
 {
     WINDOW_SHIM(WINDOW_TITLE, WW_OTHER_TABS, WH_OTHER_TABS),
-    { WWT_RESIZE,           1,  0,      529,    43,     WH_OTHER_TABS - 1,    0xFFFFFFFF,             STR_NONE },
+    MakeWidget({0, 43}, {530, 214}, WWT_RESIZE, 1),
```

Comment:
Stopped using `WH_OTHER_TABS `

Reply:
I'm not sure about `WH_OTHER_TABS`, since that's 257 and we need 214. However, `WW_OTHER_TABS` is 530, so that _would_ actually be a perfect fit. Thanks.